Rakefile

```diff
@@ -2,7 +2,7 @@ require "foodcritic/version"
 require "bundler"
 require "rake/testtask"
 
-task :default => [:man, :install, :rubocop, :test, :features]
+task :default => [:man, :install, :test, :features, :rubocop]
```

Comment:
yeah it annoys the crap out of me to have rubocop fail first because fixing that is usually mindless.